src/jit/codegenarm64.cpp

```diff
@@ -5778,7 +5777,8 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
                                                // as that is how much stack is allocated for this LclVar
                 isHfa          = varDsc->lvIsHfa();
                 gcPtrCount     = varDsc->lvStructGcCount;
-                structGcLayout = varDsc->lvGcLayout;
+                for(unsigned i = 0; i < gcPtrCount; ++i)
+                  gcPtrs[i] = varDsc->lvGcLayout[i];
```

Comment:
I pushed the whitespace change.  @jashook  Is there a way to locally/manually run clang-format with the JIT formatting rules?  I see the script for corefx (src/corefx/format-code.sh), but I do not see on for coreclr.